packages/wonder-blocks-dropdown/components/multi-select.test.js

```diff
@@ -50,16 +57,76 @@ describe("MultiSelect", () => {
         opener.simulate("mouseup");
         opener.simulate("click");
         expect(select.state("open")).toEqual(true);
+        expect(onOpen).toHaveBeenCalledTimes(1);
+        expect(onClose).toHaveBeenCalledTimes(0);
+
+        // Close select with mouse
```

Comment:
This feels like a separate test case.

Reply:
Try to keep each test case to a single expectation.

For example,

`opens the select on mouse click`
`opens the select on space`
`opens the select on enter`
`calls onOpen when opened`
`calls OnClose when closed`

It makes debugging test failures and correcting behaviour much easier that way.

Reply:
I don't think it's necessary to check the callbacks for al the different ways the component can be opened and closed.